nixos/modules/programs/ssh.nix

```diff
@@ -75,9 +72,7 @@ in
 
       hostKeyAlgorithms = mkOption {
         type = types.listOf types.str;
-        default = [
-          "+ssh-dss"
```

Comment:
Needs a changelog though.

Reply:
I don’t think it does, because as I posted above, a previous commit removed the patch that made this work in the first place.

Reply:
I will test this, though. It’s possible that the patch was applied upstream rather than just being dropped here.

Reply:
Since it's already broken I don't think the changelog is a strict requirement. It should have been for the back-compat breaking change though :)